packages/ui/tailwind-utils-config/components/index.ts

```diff
@@ -11,5 +11,6 @@ export { default as linkStyles } from './link'
 export { default as avatarStyles } from './avatar'
 export { default as cardStyles } from './card'
 export { default as inputStyles } from './input'
+export { default as textareaStyles } from './textarea'
```

Comment:
Scope for enhancement: 
Should we update every component as named export? So that we can avoid `default as` in all exports.

cc: @abhinavrastogi-harness @knagurski

Reply:
agreed, that would be cleaner. we are not using the component directly anywhere anyay

Reply:
It's not for this PR, right @praneshg239 ?

Reply:
Yes, it should be a separate PR. Not in scope of this.